esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -2691,12 +2692,14 @@ void EsdlBindingImpl::handleJSONPost(CHttpRequest *request, CHttpResponse *respo
         Owned<IPropertyTree> contentTree = createPTreeFromJSONString(content.str());
         if (contentTree)
         {
-            StringBuffer reqestName(methodName);
-            reqestName.append("Request");
+            StringBuffer requestName;
+            if (strcmp(methodName, "Ping") == 0)
```

Comment:
line 580 performs a stricmp on the methodname, let's pick one for consistency (obviously performance and functionality should be the deciding factor)

Reply:
@rpastrana changed to stricmp for consistency.